qMRLab.m

```diff
@@ -273,8 +277,24 @@ function MethodMenu(hObject, eventdata, handles, Method)
 for i=1:length(FileBrowserList)
     FileBrowserList(i).Visible('off');
 end
+
 FileBrowserList(MethodNum).Visible('on');
 
+scl_str = json2struct('ScalePanels.json');
```

Comment:
What's the purpose of this change in this MP2RAGE branch? Could you write some comments in the code to explain what this block is supposed to be adding?